<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinecraftWrapper</class>
 <widget class="QMainWindow" name="MinecraftWrapper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MinecraftWrapper</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 14px;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="logo" native="true">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>40</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/res/logo.png)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: url(:/res/btn-close.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="settingButton">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>36</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: url(:/res/btn-setting.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="gameStartButton">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>230</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>startGame</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameEdit">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>156</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>inputUsername</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>230</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="updatingLabel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>230</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>updating</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>settingButton</zorder>
    <zorder>logo</zorder>
    <zorder>closeButton</zorder>
    <zorder>gameStartButton</zorder>
    <zorder>usernameEdit</zorder>
    <zorder>progressBar</zorder>
    <zorder>updatingLabel</zorder>
   </widget>
   <widget class="QWidget" name="settingWidget" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>201</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
      <property name="spacing">
       <number>22</number>
      </property>
      <property name="leftMargin">
       <number>22</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>72</height>
         </size>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>50</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>memory</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>56</y>
           <width>50</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Java</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="memoryEdit">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>110</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="javaEdit">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>56</y>
           <width>110</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="toggleContainer">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="useJourneyRadioButton">
          <property name="text">
           <string>useJourneyMap</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="useXaerosRadioButton">
          <property name="text">
           <string>userXaerosMap</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>settingWidget</zorder>
   <zorder>mainWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MinecraftWrapper</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
